openapi.yml

```diff
@@ -5365,56 +5366,46 @@ components:
                     description: ''
                 channels:
                     description: 'Specify the broadcast channel, such as ''SMS'' or ''EMAIL'''
-                    enum:
-                        - SMS
-                        - EMAIL
-                    type: string
-                host_filters:
-                    $ref: '#/components/schemas/NotificationsHostFilterParams'
-                    description: ''
-                target:
-                    description: 'The type of object that will receive the notification such as, ''signin'' or ''host'''
-                    enum:
-                        - HOSTS
-                        - SIGNINS
-                    type: string
-        NotificationsSigninFilterParams:
-            description: ''
-            required: []
-            type: object
-            properties:
-                location_ids:
-                    description: ''
                     type: array
                     items:
-                        type: integer
-                signed_in_before:
-                    format: date-time
-                    description: ''
-                    type: string
-                signed_in_after:
-                    format: date-time
-                    description: ''
-                    type: string
-                signin_ids:
-                    description: ''
+                        type: string
+                targets:
+                    description: '''SIGNINS'' and/or ''HOSTS'''
                     type: array
                     items:
-                        type: integer
-                is_signed_out:
+                        type: string
+                location_filters:
+                    $ref: '#/components/schemas/NotificationsLocationFilterParams'
                     description: ''
-                    type: boolean
-        NotificationsHostFilterParams:
+            example:
+                message: Get to the chopper!
+                signin_filters:
+                    signin_ids:
+                        - 39
+                        - 15
+                    is_signed_out: true
+                channels:
+                    - SMS
+                    - EMAIL
+                targets:
+                    - HOSTS
```

Comment:
Question: do you have a convention to use singular or plural for these "enums"? Or this is optional?

Reply:
I think it can be one, or more of a "notifiable". I'm not sure it can qualify as an enum, because its more of an array of enums, but this type of structure isn't present in apicurio, so I made it into an array of strings.

Reply:
The reason I left it as an array is that it allows us to add other objects in the future that can receive a notification via a channel. For example if we want to send a notification to a PreRegistration, all we have to do is add a `pre_registration_filters` where we add the criteria such as ids, or any other filters, and add a target of `PREREGISTRATION` to activate that filter. 

In this case, we can infer the hosts from the signins, so I didn't add hosts_filters, but if we want to filter hosts, we can add that key in the future

Reply:
Just a point here, we'll NEVER add the word "preregistration" anywhere.
We're calling them "invites" in v3